.github/workflows/test.yml

```
@@ -87,9 +87,46 @@ jobs:
+      - name: Pull Development Image
+        run: docker pull ${{ env.DOCKER_REGISTRY_PATH }}/${{ inputs.project_name }}-development:${{ inputs.project_version }}
+      - name: Build Python Bindings
+        run: make build-packages-python-standalone
+      - name: Upload Python Bindings
+        uses: actions/upload-artifact@v4
+        with:
+          name: python-bindings # TBM: use a different namespace than in build-packages for now.
+          path: packages/python/
+      - id: python-versions
+        name: Export Python Versions to Environment
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:36: Double quote to prevent globbing and word splitting

(shellcheck)